shuup/core/management/commands/migrator.py

```diff
@@ -0,0 +1,113 @@
+from django.apps import apps as installed_apps
+from django.db import connections, router
+from mirage.crypto import Crypto
+from tqdm import tqdm
+
+
+class MirageMigrator:
```

Comment:
We need a Migrator class capable of setting a id field as a parameter for handling the child classes of ServiceProvider (which doesn't have an id field). The mirage Migrator class doesn't allow that.